src/js/accordian-input.js

```diff
@@ -0,0 +1,78 @@
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+  HOME: 36,
+  END: 35,
+  LEFT: 37,
+  UP: 38,
+  RIGHT: 39,
+  DOWN: 40
+};
+
+function getTarget(target) {
+  const $inputTarget = $(target);
+  const collapseTarget = $inputTarget.attr('href');
+  const $target = $(collapseTarget);
+
+  return $target;
+}
+
+function toggleCollapse(target) {
+  const $target = getTarget(target);
+  $target.collapse('toggle');
+}
+
+function openCollapse(target) {
+  const $target = getTarget(target);
+  $target.collapse('show');
+}
+
+function closeCollapse(target) {
+  const $target = getTarget(target);
+  $target.collapse('hide');
+}
+
+function goToFirst(target) {
+  const $target = getTarget(target);
+  const $parent = $($target.data('parent'));
+  const $first = $parent.find('[data-toggle=collapse]').first();
+
+  $first.focus();
+}
+
+function goToLast(target) {
+  const $target = getTarget(target);
+  const $parent = $($target.data('parent'));
+  const $last = $parent.find('[data-toggle=collapse]').last();
+
+  $last.focus();
+}
+
+$('[data-toggle=collapse]').keydown((e) => {
+  switch (e.which) {
+    case KEYCODES.SPACE:
+      e.preventDefault();
+      toggleCollapse(e.target);
+      break;
+    case KEYCODES.RIGHT:
+    case KEYCODES.DOWN:
```

Comment:
Directional key actions should navigate to a previous or next accordion header according to the keyboard interaction spec, not open/close the collapsible element (https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction)